docs/installation/windows.md

```diff
@@ -16,6 +16,62 @@ There is a shortcut placed in the Start Menu upon installation.
 > Note: On Windows 8.1 / Windows 2012R2, ensure Visual C++ Redistributable 
 > for VS2015 is installed from [here](http://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe):
 
+# Deploying on Nano Server
+
+These instructions assume that Windows PowerShell is running on the Nano Server image and that it has been generated by the [Nano Server Image Builder](https://technet.microsoft.com/en-us/windows-server-docs/get-started/deploy-nano-server).
+Nano Server is a "headless" OS and deployment of PowerShell Core binaries can happen in two different ways:
+
+1. Offline - Mount the Nano Server VHD and unzip the contents of the zip file to your chosen location within the mounted image.
+2. Online - Transfer the zip file over a PowerShell Session and unzip it in your chosen location.
+
+In both cases, you will need the Windows 10 x64 Zip release package and will need to run the commands within an "Administrator" PowerShell instance.
+
+## Offline Deployment of PowerShell Core
+
+1. Use your favorite zip utility to unzip the package to a directory within the mounted Nano Server image.
+2. Unmount the image and boot it
+3. Connect to the inbox instance of Windows PowerShell
+3. Follow the instructions to create a remoting endpoint using the "Executed by another instance of PowerShell on behalf of the instance that it will register" technique and reconnect to the new remoting endpoint on your Nano Server instance. In this case, the "absolute path the the instance's `$PSHOME`" is the directory where it was unzipped.
+
+## Online Deployment of PowerShell Core
+
+The following steps will guide you through deployment of PowerShell Core to a running instance of Nano Server and configuration of its remote endpoint.
+
+1. Connect to the inbox instance of Windows PowerShell
+
+```powershell
+$session = New-PSSession -ComputerName <Nano Server IP address> -Credential <An Administrator account on the system>
+```
+
+2. Copy the file to the Nano Server instance
+
+```powershell
+Copy-Item <local PS Core download location>\powershell-6.0.0-<version>-win10-x64.zip c:\ -ToSession $session
+```
+
+3. Enter the session
+
+```powershell
+Enter-PSSession $session
+```
+
+4. Create a target directory
+
+```powershell
+# Insert the appropriate version
+$tgtDir = "C:\PowerShellCore_<version>"; mkdir $tgtDir
+```
+
+5. Extract the Zip file
+
+```powershell
+[system.io.compression.zipfile]::ExtractToDirectory("c:\powershell-6.0.0-alpha.9-win10-x64.zip","$tgtDir")
+```
+
+6. Follow the instructions to create a remoting endpoint using the "Executed by another instance of PowerShell on behalf of the instance that it will register" technique and reconnect to the new remoting endpoint on your Nano Server instance. In this case, the "absolute path the the instance's `$PSHOME`" is `$tgtDir`
```

Comment:
This paragraph is very similar to the one above. I suggest the "offline deployment" and "online deployment" sections don't contain the part about configuring remote endpoint. But in the Registration section, at "Executed by another instance of PowerShell on behalf of the instance that it will register", add one line saying that this is what you should use to configure endpoint on NanoServer, and also put the note `NOTE: The remoting registration script will restart WinRM, so $session will terminate immediately after the script is run` there.